posthog/api/test/test_stickiness.py

```diff
@@ -480,7 +480,7 @@ def test_compare(self):
             )
             response = stickiness_response["result"]
             self.assertEqual(response[0]["data"], [2, 1, 1, 0, 0, 0, 0, 0])
-            self.assertEqual(response[1]["data"], [3, 0, 0, 0, 0, 0, 0, 0])
+            self.assertEqual(response[1]["data"], [0, 0, 0, 0, 0, 0, 0, 0])
```

Comment:
so the test was wrong because the date was picking the wrong ceiling/floor?

Could we put matches either side of the correct date boundary so you get a fail if that boundary changes? Or is that covered elsewhere?

Reply:
Frankly I don't see at all why that was non-zero before, because the data generation function includes no events before the start of the "current" period (January 1st). Added `@snapshot_clickhouse_queries` to dig into this.